### PR TITLE
zh-translation:zh\docs\tasks\security\authentication

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by yanrongshi** [comment]
> 
Could you merge it for me, please

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+keywords: [security,authentication,jwt,route]
+owner: istio/wg-security-maintainers
+test: yes
+status: Experimental
+---
+
+本任务向您展示如何实现基于 Istio 入口网关上的 JWT 声明路由请求，来使用请求身份认证
+和虚拟服务。
+
+注意:该特性只支持 Istio 入口网关，并且需要使用请求身份验证和虚拟
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:14]: 用中文冒号。

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+{{< boilerplate experimental-feature-warning >}}
+
+## 开始之前{#before-you-begin}
+
+* 理解 Istio [身份认证策略](/zh/docs/concepts/security/#authentication-policies)和[虚拟服务](/zh/docs/concepts/traffic-management/#virtual-services)相关概念。
+
+* 使用[Istio 安装指南](/zh/docs/setup/install/istioctl/)安装 Istio。
+
+* 在一个命名空间中,部署一个 `httpbin` 工作负载，例如 `foo`，
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:25]: 中文符号。

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+## 清除{#cleanup}
+
+* 移除 `foo` 命名空间：
+
+    {{< text bash >}}
+    $ kubectl delete namespace foo
+    {{< /text >}}
+
+* 移除请求身份认证：
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:169]: 这个可以不翻译。

**Review by kebe7jun — COMMENTED** [review]
一些 crd 可以不翻译。

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+注意：该特性只支持 Istio 入口网关，并且需要使用请求身份验证和虚拟
+服务来根据 JWT 声明进行正确的验证和路由。
+
+{{< boilerplate experimental-feature-warning >}}
+
+## 开始之前{#before-you-begin}
+
+* 理解 Istio [身份认证策略](/zh/docs/concepts/security/#authentication-policies)和[虚拟服务](/zh/docs/concepts/traffic-management/#virtual-services)相关概念。
+
+* 使用[Istio 安装指南](/zh/docs/setup/install/istioctl/)安装 Istio。
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:23]: need a space between zh and en word

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+      selector:
+        matchLabels:
+          istio: ingressgateway
+      jwtRules:
+      - issuer: "testing@secure.istio.io"
+        jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+    EOF
+    {{< /text >}}
+
+    这个请求身份验证将在 Iitio 网关上启用 JWT 校验，以便验证过的 JWT 声明稍后可以在虚拟服务中用于路由功能。
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:72]: fix Iitio to Istio

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+    虚拟服务使用保留的消息头 `"@request.auth.claims.groups"` 来匹配 JWT 声明中的 `groups`。
+    前缀的 `@` 表示它与来自 JWT 验证的元数据匹配，而不是与 HTTP 消息头匹配。
+    JWT 支持字符串类型的声明、字符串列表和嵌套声明。使用 `.` 作为嵌套声明名称的分隔符。
+    例如， `"@request.auth.claims.name.givenName"` 匹配嵌套声明 `name` 和 `givenName`。 使用 `.` 字符作为声明
+    名称在当前是不被支持的。
+
+## 基于 JWT 声明验证入口路由{#validating-ingress-routing-based-on-JWT-claims}
+
+1. 验证入口网关返回没有 JWT 的 HTTP 404代码：
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:115]: need a space between number and words

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+    {{< text bash >}}
+    $ curl -s -I "http://$INGRESS_HOST:$INGRESS_PORT/headers"
+    HTTP/1.1 404 Not Found
+    ...
+    {{< /text >}}
+
+    您还可以创建授权策略，以便在缺少 JWT 时使用 HTTP 403代码显式拒绝请求。
+
+1. 验证入口网关返回带有无效 JWT 的 HTTP 401代码：
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:125]: need a space between number and words

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+    {"exp":3537391104,"groups":["group1","group2"],"iat":1537391104,"iss":"testing@secure.istio.io","scope":["scope1","scope2"],"sub":"testing@secure.istio.io"}
+    {{< /text >}}
+
+    {{< text bash >}}
+    $ curl -s -I "http://$INGRESS_HOST:$INGRESS_PORT/headers" -H "Authorization: Bearer $TOKEN_GROUP"
+    HTTP/1.1 200 OK
+    ...
+    {{< /text >}}
+
+1. 验证入口网关，返回了带有有效 JWT 的 HTTP 404代码，但不包含 `groups: group1` 声明：
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:148]: need a space between number and words

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+## Cleanup
+
+* Remove the namespace foo:
+
+    {{< text bash >}}
+    $ kubectl delete namespace foo
+    {{< /text >}}
+
+* Remove the request authentication:
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:169]: Need to translate this part into zh

**Review by Xunzhuo — CHANGES_REQUESTED** [review]

**Comment by yanrongshi** [comment]
hey,What should I do next？

**Comment by Xunzhuo** [comment]
> hey,What should I do next？

Fix the issues that reviewers commented on your PR

**Comment by yanrongshi** [comment]
I have changed it. Please help me check it. Thank you very much

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+{{< boilerplate experimental-feature-warning >}}
+
+## 开始之前{#before-you-begin}
+
+* 理解 Istio [身份认证策略](/zh/docs/concepts/security/#authentication-policies)和[虚拟服务](/zh/docs/concepts/traffic-management/#virtual-services)相关概念。
+
+* 使用 [Istio 安装指南](/zh/docs/setup/install/istioctl/)安装 Istio 。
+
+* 在一个命名空间中，部署一个 `httpbin` 工作负载，例如 `foo` ，
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:25]:
```suggestion
* 在 `foo` 命名空间中，部署一个 `httpbin` 工作负载 ，
```

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+            port:
+              number: 8000
+            host: httpbin
+    EOF
+    {{< /text >}}
+
+    虚拟服务使用保留的消息头 `"@request.auth.claims.groups"` 来匹配 JWT 声明中的 `groups` 。
+    前缀的 `@` 表示它与来自 JWT 验证的元数据匹配，而不是与 HTTP 消息头匹配。
+    JWT 支持字符串类型的声明、字符串列表和嵌套声明。使用 `.` 作为嵌套声明名称的分隔符。
+    例如， `"@request.auth.claims.name.givenName"` 匹配嵌套声明 `name` 和 `givenName` 。 使用 `.` 字符作为声明
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:110]:
```suggestion
    例如， `"@request.auth.claims.name.givenName"` 匹配嵌套声明 `name` 和 `givenName` 。 当前不支持使用 `.` 字符作为声明名称。
```

**Review by Xunzhuo — APPROVED** [review]

File: content/zh/docs/tasks/security/authentication/jwt-route/index.md [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+              number: 8000
+            host: httpbin
+    EOF
+    {{< /text >}}
+
+    虚拟服务使用保留的消息头 `"@request.auth.claims.groups"` 来匹配 JWT 声明中的 `groups` 。
+    前缀的 `@` 表示它与来自 JWT 验证的元数据匹配，而不是与 HTTP 消息头匹配。
+    JWT 支持字符串类型的声明、字符串列表和嵌套声明。使用 `.` 作为嵌套声明名称的分隔符。
+    例如， `"@request.auth.claims.name.givenName"` 匹配嵌套声明 `name` 和 `givenName` 。 当前不支持使用 `.` 字符作为声明名称。
+    名称在当前是不被支持的。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/jwt-route/index.md:111]:
```suggestion
```

**Review by kebe7jun — CHANGES_REQUESTED** [review]

**Review by kebe7jun — APPROVED** [review]

**Comment by yanrongshi** [comment]
Thanks for your help ! @Xunzhuo ，@kebe7jun

**Pull request merged**